### PR TITLE
Adds a pink scarf

### PR DESCRIPTION
**Pull request by Hyacinth-OR**
Pink is an excellent accent color to contrast with a dark outfit, or to complement a lighter, more summery getup.

Scarves are great, here's one in a great color.

:cl:
add: Added a pink scarf to vendomats
/:cl:

### CODE REVIEW

**Review by Cyberboss — APPROVED** [review]

**Comment by Therapist1** [comment]
I just wonder how many people will flip over a pink fuckin scarf

**Comment by Aloraydrel** [comment]
People who can't play will get upset

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -128,7 +133,7 @@
 /obj/item/clothing/neck/scarf/orange
 	name = "orange scarf"
 	icon_state = "scarf"
-	color = "#C67A4B" //orange
+	color = "#C67A4B" //Orange
```
Comment (pubby) [inline code/modules/clothing/neck/_neck.dm]: hm

**Comment by sawrge** [comment]
fetish content

**Comment by Mark9013100** [comment]
slibbery slobe

**Comment by Anonmare** [comment]
Powercreep

File: code/modules/clothing/neck/_neck.dm (hunk 1 above)
Comment (Anonmare) [reply code/modules/clothing/neck/_neck.dm]: No player references

**Pull request merged**